backend/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nr-quickstart-typescript",
+  "name": "quickstart-openshift",
```

Comment:
should we update this one to `quickstart-openshift-backend` in this one, or may be a separate PR?

Reply:
If we're not publishing it shouldn't matter.  I renamed to `backend`, which is appropriately generic.